Project Commit 1

z-fonz opened the pull request:
I had attempted to make it so I was able to edit the data inside the testbank database. In models.py I had changed the databases a little bit where instead of db.model I replaced it with Table, however if this change wasn’t needed, myself or anyone else can feel free to change it back. I had also added an app route in routes.py. Feel free to comment on what looks good or what doesn’t and I will change accordingly.

Commit:
Project Commit 1

I had attempted to make it so I was able to edit the data inside the testbank database. In models.py I had changed the databases a little bit where instead of db.model I replaced it with Table, however if this change wasn’t needed, myself or anyone else can feel free to change it back. I had also added an app route in routes.py. Feel free to comment on what looks good or what doesn’t and I will change accordingly.

Commit:
Project Commit 2

Due to conflicts with the previous pull request, I had changed the Table back to db.model in regards to all of the databases inside the models.py file.

eduardo-garcia16 commented:
Closing pull request as you already pushed your routes.py file, and the models.py needed no real change.

eduardo-garcia16 committed:
Merge branch 'master' into editDatabase

Commit:
Update .DS_Store

Commit:
Update .DS_Store

Commit:
Merge branch 'editDatabase' of https://github.com/eduardo-garcia16/CUS1166_Project into editDatabase

Commit:
Troubleshoot Commit

Done because I was having trouble getting my branch up to date with changes from the master branch

Commit:
.DS_Store banished!

Commit:
.DS_Store banished!

Commit:
Merge branch 'master' into editDatabase

Commit:
Sprint 1.5 Commit

The following changed have been made:
Created a databse to hold a teacher user.
Made changes to my route introduced in sprint 1 (edit_question)
Changed name of the specific database "testbank" to "globaltestbank".
Create new databases: teachertestbank, teacherUser.
Added route for teacherUser.
